packages/bible-book-names-intl/src/data/translations/pt.json

```diff
@@ -0,0 +1,520 @@
+        "Sm"
+      ],
+      "startNumber": 0
+    },
+    "20": {
+      "name": "Provérbios",
+      "shortNames": [
+        "Pv",
+        "Prov",
+        "Pro"
+      ],
+      "startNumber": 0
+    },
+    "21": {
+      "name": "Eclesiastes",
+      "shortNames": [
+        "Ec"
+      ],
+      "startNumber": 0
+    },
+    "22": {
+      "name": "Cânticos de Salomão",
+      "shortNames": [
+        "Ct"
+      ],
+      "startNumber": 0
+    },
+    "23": {
+      "name": "Isaías",
+      "shortNames": [
+        "Is"
+      ],
+      "startNumber": 0
+    },
+    "24": {
+      "name": "Jeremias",
+      "shortNames": [
+        "Jr"
+      ],
+      "startNumber": 0
+    },
+    "25": {
+      "name": "Lamentações",
+      "shortNames": [
+        "Lm",
+        "Lam"
+      ],
+      "startNumber": 0
+    },
+    "26": {
+      "name": "Ezequiel",
+      "shortNames": [
+        "Ez"
+      ],
+      "startNumber": 0
+    },
+    "27": {
+      "name": "Daniel",
+      "shortNames": [
+        "Dn",
+        "Dan"
+      ],
+      "startNumber": 0
+    },
+    "28": {
+      "name": "Oseias",
+      "shortNames": [
+        "Os"
+      ],
+      "startNumber": 0
+    },
+    "29": {
+      "name": "Joel",
+      "shortNames": [
+        "Jl",
+        "Joe"
+      ],
+      "startNumber": 0
+    },
+    "30": {
+      "name": "Amós",
+      "shortNames": [
+        "Am",
+        "Amo"
+      ],
+      "startNumber": 0
+    },
+    "31": {
+      "name": "Obadias",
+      "shortNames": [
+        "Ob",
+        "Oba"
+      ],
+      "startNumber": 0
+    },
+    "32": {
+      "name": "Jonas",
+      "shortNames": [
+        "Jn"
+      ],
+      "startNumber": 0
+    },
+    "33": {
+      "name": "Miqueias",
+      "shortNames": [
+        "Mq"
+      ],
+      "startNumber": 0
+    },
+    "34": {
+      "name": "Naum",
+      "shortNames": [
+        "Na"
+      ],
+      "startNumber": 0
+    },
+    "35": {
+      "name": "Habacuque",
+      "shortNames": [
+        "Hc"
+      ],
+      "startNumber": 0
+    },
+    "36": {
+      "name": "Sofonias",
+      "shortNames": [
+        "Sf"
+      ],
+      "startNumber": 0
+    },
+    "37": {
+      "name": "Ageu",
+      "shortNames": [
+        "Ag"
+      ],
+      "startNumber": 0
+    },
+    "38": {
+      "name": "Zacarias",
+      "shortNames": [
+        "Zc"
+      ],
+      "startNumber": 0
+    },
+    "39": {
+      "name": "Malaquias",
+      "shortNames": [
+        "Ml"
+      ],
+      "startNumber": 0
+    },
+    "40": {
+      "name": "Mateus",
+      "shortNames": [
+        "Mt",
+        "Mat"
+      ],
+      "startNumber": 0
+    },
+    "41": {
+      "name": "Marcos",
+      "shortNames": [
+        "Mc"
+      ],
+      "startNumber": 0
+    },
+    "42": {
+      "name": "Lucas",
+      "shortNames": [
+        "Lc",
+        "Luc"
+      ],
+      "startNumber": 0
+    },
+    "43": {
+      "name": "João",
+      "shortNames": [
+        "Jo"
```

Comment:
Thanks for the contribution.

For the book abbreviations, the data structure require is to be unique.


I just saw Jo was used in Line 152 for Book NO18 as well.